pkg/pool-stable/contracts/StablePool.sol

```diff
@@ -395,28 +395,20 @@ contract StablePool is BaseGeneralPool, LegacyBaseMinimalSwapInfoPool, IRateProv
         internal
         virtual
         override
+        whenNotPaused
```

Comment:
Can we add this modifier on BaseLegacyPool?

Reply:
No - it's pure virtual there, so you can't add modifiers.